.gitattributes

```diff
@@ -1,4 +1,10 @@
-*               text=auto
-build/**        linguist-generated=false
-lib/**          linguist-generated=true
-doc/**          linguist-documentation
+* 						          text=auto
+.babelrc        		    diff=jsonc linguist-language=jsonc
+.devcontainer/*.json	  diff=jsonc linguist-language=jsonc
+.eslintrc        		    diff=jsonc linguist-language=jsonc
+.tsconfig.json          diff=jsonc linguist-language=jsonc
+.vscode/*.json			    diff=jsonc linguist-language=jsonc
+build/**        		    linguist-generated=false
+doc/**          		    linguist-documentation
+lib/**          		    linguist-generated=true
+OWNERS          		    diff=json5 linguist-language=json5
```

Comment:
I am thinking about switching from tabs to spaces&hellip;

It's not going well, as you can see&hellip;